src/models/audio.rs

```diff
@@ -22,8 +26,22 @@ pub enum AudioPlayback {
     Bought(i64),
 }
 
+/*
+   How to use a callback
+
+   let error_callback = Closure::once(Box::new(move |_: web_sys::Event| {
+       // Error playing audio, fall back to TTS
+       tts_play(&tts_text);
+    }));
+
+    // Attempt to add the error event listener
+    let _ = audio.add_event_listener_with_callback("error", error_callback.as_ref().unchecked_ref());
+*/
```

Comment:
consider changing this to a doc comment: ///

Reply:
no